mmv1/templates/terraform/encoders/compute_instance_group_membership.go.erb

```diff
@@ -0,0 +1,23 @@
+<%# The license inside this block applies to this file.
+  # Copyright 2017 Google Inc.
+  # Licensed under the Apache License, Version 2.0 (the "License");
+  # you may not use this file except in compliance with the License.
+  # You may obtain a copy of the License at
+  #
+  #     http://www.apache.org/licenses/LICENSE-2.0
+  #
+  # Unless required by applicable law or agreed to in writing, software
+  # distributed under the License is distributed on an "AS IS" BASIS,
+  # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  # See the License for the specific language governing permissions and
+  # limitations under the License.
+-%>
+// Instance Group is a URL parameter only, so replace self-link/path with resource name only.
+if err := d.Set("instance_group", tpgresource.GetResourceNameFromSelfLink(d.Get("instance_group").(string))); err != nil {
```

Comment:
I'm a little surprised to see a `d.Set` like this in the encoder. Did you copy this from somewhere?

Reply:
From `network_endpoint` resource, as it works in the similar way as `instance_group_membership`. Both are using fine-grained resources.